Add onelogin and okta SAML docs (#1108)
CloudFire-LLC/cloudfire: docs/docs/authenticate/saml/onelogin.mdx

---
title: OneLogin
sidebar_position: 2
description: Firezone's WireGuard based remote access platform supports using OneLogin as a single sign-on provider using a SAML 2.0 connector.
---

:::note
This guide assumes you have completed the prerequisite steps
outlined [here](/authenticate/saml).
:::

> Configure OneLogin as an identity provider using a SAML 2.0
> connector so users can authenticate with their OneLogin credentials.
> Follow the steps below to activate the integration.

Reviewer: ```suggestion
Firezone supports Single Sign-On (SSO) using OneLogin through the generic SAML 2.0 connector. This guide will walk you through how to configure the integration.
```

## Create a SAML connector

In the OneLogin admin portal, add an app under the application tab.
Select `SAML Custom Connector (Advanced)` and provide the appropriate
configuration settings under the under the configuration tab.

The following fields should be filled out on this page:

| Setting                | Value                                                                  |

> |---                      |---                                                                     |

Reviewer: ```suggestion
|---                     |---                                                                     |
```

| SAML initiator         | `Service Provider`                                                     |
| SAML signature element | `Both`                                                                 |

[OneLogin's docs](https://onelogin.service-now.com/support?id=kb_article&sys_id=912bb23edbde7810fe39dde7489619de&kb_category=93e869b0db185340d5505eea4b961934)
provide a good overview of each field's purpose. In all the fields above, replace `firezone.company.com`
with your deployment's external URL.

Once complete, save the changes and download the SAML metadata document
found unde the `More Actions` dropdown.

## Add SAML identity provider to Firezone

In the Firezone portal, add a SAML identity provider under the Security tab
by filling out the following information:

| Setting   | Value      | Notes                                                                                                                                 |
|---        |---         |---                                                                                                                                    |

> | Config ID | `onelogin` | Firezone uses this value to construct endpoints required in the SAML authentication flow (e.g., receiving assertions, login requests) |

Reviewer: ```suggestion
| Config ID | `onelogin` | Firezone uses this value to construct endpoints required in the SAML authentication flow (e.g., receiving assertions, login requests).|
```
